freshmaker/events.py

```diff
@@ -269,15 +269,17 @@ def __init__(self, msg_id, task_id, task_state, **kwargs):
 
 
 class ErrataBaseEvent(BaseEvent):
-    def __init__(self, msg_id, advisory, **kwargs):
+    def __init__(self, msg_id, advisory, freshmaker_event_id=None, **kwargs):
         """
         Creates new ErrataBaseEvent.
 
         :param str msg_id: Message id.
         :param ErrataAdvisory advisory: Errata advisory associated with event.
+        :param freshmaker_event_id: Freshmaker event id on which this event is based on.
         """
         super(ErrataBaseEvent, self).__init__(msg_id, **kwargs)
         self.advisory = advisory
+        self.freshmaker_event_id = freshmaker_event_id
```

Comment:
Could you please delete the line `self.freshmaker_event_id = freshmaker_event_id` in `ManualRebuildWithAdvisoryEvent.__init__` since this is now handled here (the base class).